### ISSUE TITLE
Probably not a Bug really, but php artisan migrage throws an error depending up MySQL

### ISSUE THREAD

**Issue by sscotti** [rappasoft/laravel-boilerplate#1544]
Have NameCheap as a hosting service for some projects.

Syntax error or access violation: 1071 Specified key was too long occurs when doing:  php artisan migrate.

Fix is to add the following to the AppServiceProvider.php

use Illuminate\Support\Facades\Schema;

    public function boot()
    {
        Paginator::useBootstrap();
        Schema::defaultStringLength(191);
    }
}

**Comment by rappasoft** [comment]
That bug has to be 5 or more years old at this point (Laravel 5.4 I believe). We had that in our AppServiceProvider for the longest time. It was removed when we figured no one would be running such old versions of MySQL anymore (< 5.7.7) the current latest version is 8. If you're getting this error you are on at least 5.6 which is no longer supported:

<img width="707" alt="Screen Shot 2021-08-29 at 12 10 37" src="https://user-images.githubusercontent.com/6026504/131257351-df4c463b-d700-4718-9935-dd4d157b4316.png">
